CHANGELOG.md

```diff
@@ -1,5 +1,122 @@
 # Versions
 
+## v1.0.0-beta.3
+
+_Sep 3, 2025_
+
+### General changes
+
+- **Breaking change:** Base UI event details.
+  Custom event callbacks provide BaseUIEventDetails object as their second parameter.
+  This object contains the source event, reason and methods to customize the behavior (where applicable).
+  (#2382) by @atomiks
```

Comment:
Would be good to show the change in signature from before, mainly for `onOpenChange`